### PR TITLE
JBPM-6888: Remove signal and abort actions from process instance deta…

### PR DESCRIPTION
**Pull request by nmirasch**
…ils page when visiting from dashboard

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac could you take a look? thanks!

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java [hunk 1]
```diff
@@ -129,9 +135,26 @@ public String getTitle() {
 
     @OnStartup
     public void onStartup(final PlaceRequest place) {
+        signalProcessInstanceAction = new PrimaryActionMenuBuilder(constants.Signal(),
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java]: perhaps use set methods in here?

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenterTest.java [hunk 2]
```diff
@@ -71,6 +73,11 @@
     @Mock
     private ProcessService processService;
 
+    @Mock
+    PrimaryActionMenuBuilder signalProcessInstanceAction;
+
+    @Mock
+    PrimaryActionMenuBuilder abortProcessInstanceAction;
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenterTest.java]: new line space? :)

**Comment by nmirasch** [comment]
@cristianonicolai updated

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenterTest.java (hunk 2 above)
Comment (nmirasch) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenterTest.java]: :D sure

**Pull request merged**